packages/shell-api/src/collection.ts

```diff
@@ -608,7 +608,7 @@ export default class Collection extends ShellApiClass {
       this._mongo._internalState.context.print
     );
     assertArgsDefined(docs);
-    const d: Document[] = Array.isArray(docs) ? docs : [docs];
+    const d: Document[] = Array.isArray(docs) ? docs.map((doc) => ({ ...doc })) : [{ ...docs }];
```

Comment:
why not keeping the variable name something more readable than 1 letter?

Reply:
We could find a better name, i just updated the value.

Reply:
true that, it was like that already :)

Reply:
Updated to make the name more clear.